serializr.js

```diff
@@ -167,8 +167,30 @@
             }
         }
 
+				// Ugly way to get the parameter names since they aren't easily retrievable via reflection
+				const STRIP_COMMENTS = /((\/\/.*$)|(\/\*[\s\S]*?\*\/))/mg;
+				const ARGUMENT_NAMES = /([^\s,]+)/g;
+				function getParamNames(func) {
```

Comment:
Added a requirement of using alias() when serializing constructor variables in Typescript so we don't run into a problem with minimization.